debian/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:jessie
+RUN apt-get update && apt-get install -qqy curl python2.7 gcc python-dev python-setuptools apt-transport-https lsb-release  && \
```

Comment:
ok. it wasn't clear what you were getting with just the lsb-release bit.

anyway, it seems to be just python2.7 only since thats installed  with setuptools, -dev.
(and yeah, the rest is for compiled crcmod; pls verify it to)